tendrl/monitoring_integration/flows/delete_resource_from_graphite/__init__.py

```diff
@@ -30,179 +30,186 @@ def get_data_dir_path(self):
-                                             1)[1].split(":",
-                                                         1)[1].replace("/",
-                                                                       '\|')
-            vol_name = resource_name.split("|", 1)[0]
-            archive_path = os.path.join(WHISPER_PATH, "clusters",
-                                        str(cluster_id),
-                                        "archive", "bricks")
-            if not os.path.exists(archive_path):
-                os.system("mkdir -p " + str(archive_path))
-            resource_folder_name = str(brick_name) + "_" + \
-                str(datetime.datetime.now().isoformat())
-            archive_path = os.path.join(archive_path, resource_folder_name)
-            resource_path = os.path.join(WHISPER_PATH, "clusters",
-                                         str(cluster_id), "nodes",
-                                         str(host_name), "bricks",
-                                         str(brick_name))
-            brick_path = os.path.join(
-                WHISPER_PATH,
-                "clusters",
-                str(cluster_id),
-                "nodes",
-                str(host_name),
-                "bricks",
-                resource_name.split("|", 1)[1].split(":", 1)[1].replace("/", "|")
-            )
-            if os.path.exists(brick_path):
-                ret_val = os.system("mv " + str(resource_path) + " " +
-                          str(archive_path))
-                if ret_val is 0:
-                    logger.log("info", NS.get("publisher_id", None),
-                              {'message': "Brick " + str(resource_name) + 
-                                       "deleted from graphite"})
-                else:
-                     logger.log("error", NS.get("publisher_id", None),
-                               {'message': "Brick " + str(resource_name) + 
-                                       "deletion from graphite failed"})
+                                "deletion from graphite failed"})
+
+    def delete_host_details(self, cluster_id, resource_name, whisper_path):
+        host_name = resource_name.replace(".", "_")
+        volume_affected_list = []
+        volume_key = os.path.join("clusters", str(cluster_id), "Volumes")
+        volume_list = create_dashboards.get_resource_keys("", volume_key)
+        for volume_id in volume_list:
+            flag = False
+            single_volume_key = os.path.join(volume_key, volume_id,
+                                             "Bricks")
+            subvolume_list = create_dashboards.get_resource_keys("", single_volume_key)
+            for subvolume in subvolume_list:
+                subvolume_brick_key = os.path.join(single_volume_key,
+                                                   subvolume)
+                brick_list = create_dashboards.get_resource_keys("", subvolume_brick_key)
+                if flag:
+                    flag = False
```

Comment:
Where is flag being set as `True`?

Reply:
@shtripat https://github.com/Tendrl/monitoring-integration/pull/154/files#diff-df788ce9a7b7a09945c3cd9e735da7d5R159

Reply:
Ah my bad didnt check down enough :) Ack